gen/generate_test.go

```diff
@@ -79,6 +79,62 @@ func TestGenerateWithRelativePaths(t *testing.T) {
 	}
 }
 
+func TestGenerateWithHyphenPaths(t *testing.T) {
+	outputDir, err := ioutil.TempDir("", "thriftrw-generate-test")
```

Comment:
let's use `tmp/thriftrw-generate-test` so we don't clutter the existing directory

Reply:
whoops, I misread the API!